hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -138,9 +141,11 @@ public boolean dropTable(TableIdentifier identifier, boolean purge) {
         dropTableData(ops.io(), lastMetadata);
       }
 
+      LOG.info("Table is dropped [{}]", identifier);
```

Comment:
How about "Dropped table: {}" instead of using a passive form? We also tend to use `:` instead of brackets in messages.

Reply:
Done, I will also remove every '[]' - I tend to use them because once spent 2 days trying to figure out why my code is not able to find table 'sneakytable', where the log cleanly shows that 'sneakytable ' is in the listing (notice the missing ' ', which is hardly noticeable in the logs if the beginning/end of the variable is not marked clearly, and it is the end of the line :))

Reply:
I too have endured this pain of whitespace at the start/end of a string like this causing all kinds of problems and being really hard to spot as the cause!

Reply:
For concerns like this, I prefer to have a `quote` method that handles special characters. That way you get a readable error message when everything is normal, but the space problem is highlighted when it needs to be. So `ident.quote()` would return `db.table` normally and `db."table "` if there is a trailing space.